### PR TITLE
[RW-1288] Changing load_data_frame to run BigQuery queries directly.

### PR DESCRIPTION
**Pull request by danqrodney**
Adding load_annotations.

Bumping version to workbench api_v1_10.

Getting rid of materialize_cohort, load_data_table.

### CODE REVIEW

File: py/aou_workbench_client/data.py [hunk 1]
```diff
@@ -50,24 +53,23 @@
...
     [Person.gender_concept_id, Person.person_id]
-  :param page_size: the maximum number of result rows to fetch in a single API request when 
-    retrieving results; defaults to 1000.
   :param debug: true if debug request and response information should be displayed; defaults to 
     false.  
-  :return a generator of dictionaries representing the results to the query
+  :return a Pandas data frame representing the results of the query
 """
-def load_data_table(cohort_name, table, columns=None, concept_set_name=None, concept_ids=None,
+def load_data_frame(cohort_name, table, columns=None, concept_set_name=None, concept_ids=None,
```
Comment (calbach) [inline py/aou_workbench_client/data.py]: nit: naming seems slightly asymmetric with `load_data_frame()` and `load_annotations` (both returning data frames), should this just be called `load` or `load_data`?

File: py/aou_workbench_client/cohorts.py [hunk 2]
```diff
@@ -1,48 +1,40 @@
...
-    from materializing a cohort in the workspace containing this notebook, based
-    on the provided MateralizeCohortRequest."""
+def get_data_table_query(data_table_specification, debug=False):
+    """Returns a CdrQuery that can be used to retrieve data from BigQuery for the
+    specified data table specification.
+
+    :param data_table_specification: the DataTableSpecification to get the query for
+    :param debug: true if debug request and response information should be
+    displayed; defaults to false.
+    :return a query with the SQL and configuration to use when retrieving data
```
Comment (calbach) [inline py/aou_workbench_client/cohorts.py]: Should document the type that it returns

File: py/aou_workbench_client/cohorts.py [hunk 3]
```diff
@@ -1,48 +1,40 @@
...
     client = get_authenticated_swagger_client(debug=debug)
     cohorts_api = CohortsApi(api_client=client)
-    return cohorts_api.materialize_cohort(all_of_us_config.workspace_namespace,
-                                          all_of_us_config.workspace_id,
-                                          request=request)
+    return cohorts_api.get_data_table_query(all_of_us_config.workspace_namespace,
+                                            all_of_us_config.workspace_id,
+                                            request=data_table_specification)
 
-def materialize_cohort(request, max_results=None, debug=False):
```
Comment (calbach) [inline py/aou_workbench_client/cohorts.py]: Is it not possible to continue providing this with a deprecation notice? Or is it incompatible? I don't think it's worth doing significant work to have backwards compatibility, but the transition here might be a little bit rough, e.g. for old demo notebooks.

File: py/aou_workbench_client/cohorts.py (hunk 2 above)
Comment (danqrodney) [reply py/aou_workbench_client/cohorts.py]: Done.

File: py/aou_workbench_client/cohorts.py (hunk 3 above)
Comment (danqrodney) [reply py/aou_workbench_client/cohorts.py]: I checked in with @fonnesbeck  on #workbench-beta about this. He thought it'd be OK to make a breaking change to the client libs. Note that we're maintaining server backwards compatibilty (at least for now), so if you don't want your notebook to break, you can simply not upgrade your pyclient library. (We're not automatically updating pyclient libs yet, right?)

Once we actually have launched and have notebooks out in the wild, we will obviously need to be way more careful about maintaining backwards compatibility in client libraries, too.

**Comment by danqrodney** [comment]
Note: I had to change our Circle Dockerfile to use Python 3.6.6 to fix a problem with installing requirements. (I can't use Python 3.7.0 as the version of Pandas we're using won't work with it.)

**Comment by danqrodney** [comment]
I updated the README to reflect these changes (taking a number of things out -- we no longer deal with the underlying server API objects like FieldSet and TableQuery, since you don't really need to use them anymore.)

We were also missing documentation on the concept_set_name parameter before, which is there now.

File: py/aou_workbench_client/cohorts.py (hunk 3 above)
Comment (calbach) [reply py/aou_workbench_client/cohorts.py]: We are automatically upgrading client libraries when we recreate the cluster, i.e. on a 1w-2w interval depending on when they last signed in.

As long as it won't disrupt user testing it's probably fine (I believe they are creating new workspaces in user testing, but worth confirming). Note that we will also have cases where people are looking at the same notebook from different clusters which have different client library versions.

Comment (danqrodney) [reply py/aou_workbench_client/cohorts.py]: OK. We always have the option of not changing the client library that gets installed by workbench right away, if we want to keep the existing stuff running for longer. I'll check to make sure this doesn't mess with user testing.

File: py/README.md [hunk 4]
```diff
@@ -100,158 +101,44 @@ count order; the concepts that occur for the most participants in the CDR are re
...
-
-To instead return all the results as an in-memory Pandas DataFrame, use the convenience method
-[load_data_frame](#aou_workbench_clientdataload_data_frame).
-
-### `aou_workbench_client.data.load_data_frame`
-
-`load_data_frame` executes [load_data_table](#aou_workbench_clientdataload_data_table) and returns all the results
-as an in-memory Pandas DataFrame. Note that many API calls may need to be performed in order to retrieve all the results;
-for large cohorts this may take a while.
+Results from this method are returned as Pandas DataFrame with the columns requested. All results
```
Comment (calbach) [inline py/README.md]: nit: as *a* Pandas DataFrame

Comment (danqrodney) [reply py/README.md]: Done.

**Comment by danqrodney** [comment]
@fonnesbeck gives the thumbs up. I'm going to merge.

**Pull request merged**